estuary/models/koji.py

```diff
@@ -66,7 +66,7 @@ class ContainerKojiBuild(KojiBuild):
     """A Neo4j definition of a build that represents a container build in Koji."""
 
     original_nvr = StringProperty()
-    triggered_by_freshmaker_event = RelationshipFrom(
```

Comment:
I prefer this original name. What was the motivation for changing it?

Reply:
I thought it was more representative. I'll change it back.